src/feedback/snackbar/snackbar.jsx

```diff
@@ -0,0 +1,100 @@
+import {
+    get,
+} from 'lodash';
+import ClassNames from 'classnames';
+import MUISnackbar from '@material-ui/core/Snackbar';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    BEM_SNACKBAR,
+    UI_CLASS_NAME,
+} from '../../global/constants';
+import makeStyles from '../../styles/makeStyles';
+
+const propTypes = {
+    /**
+     * Override or extend the styles applied to the component.
+     * See [CSS API](https://material-ui.com/api/snackbar/#css) for more details.
+     */
+    classes: PropTypes.shape({}),
+    /**
+     * Assign additional class names to Mobile Stepper.
+     */
+    className: PropTypes.string,
+    /**
+     * Props applied to the [`SnackbarContent`](/api/snackbar-content/) element.
+     */
+    ContentProps: PropTypes.shape({
+        classes: PropTypes.shape({}),
+    }),
```

Comment:
Is this deliberately capitalized?

Reply:
Yes. see https://github.com/saddlebackdev/react-cm-ui/pull/310#discussion_r658151359

Reply:
Copy that.  It's MUI's fault! 😁 
Resolving.